### PR TITLE
Feature: Updated the design of the home page headers

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Motivation & Context

- Improved UI

### PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

image

### CODE REVIEW

File: src/Files.App/UserControls/Widgets/DrivesWidget.xaml [hunk 1]
```diff
@@ -8,12 +8,34 @@
 	xmlns:local="using:Files.App.UserControls.Widgets"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	mc:Ignorable="d">
-	<Grid>
+
+	<local:HomePageWidget.Resources>
```
Comment (yaira2) [inline src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: Is this out of scope?

**Comment by mdtauk** [comment]
The header should be 40 high, and the hover backplate 32 high.

Also there should be less padding for the top and bottom of the widgets, as it feels off balanced.

File: src/Files.App/UserControls/Widgets/DrivesWidget.xaml (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: I dont think so. This card shound not have the same padding & style as the default one. On hovering they will be emphasized like settings control in win11 settings.

Comment (0x5bfa) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: So it is changed margin to 12,8 or so, shouldn't be? @mdtauk

Comment (yaira2) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: What's the before and after?

Comment (mdtauk) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: image

Comment (0x5bfa) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: Before
image

After
image

Comment (0x5bfa) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: What about emphasizing border styling (behaviour is like settings button in win11 settings app)? should be removed?

Comment (yaira2) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: We haven't made any decisions on the card style so this should be limited to updating the expander style.

Comment (0x5bfa) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml]: All removed!

**Comment by 0x5bfa** [comment]
Done. Ready for review.

File: src/Files.App/UserControls/AddressToolbar.xaml [hunk 2]
```diff
@@ -416,52 +416,6 @@
 					</VisualState.Setters>
 				</VisualState>
 			</VisualStateGroup>
-			<VisualStateGroup>
```
Comment (yaira2) [inline src/Files.App/UserControls/AddressToolbar.xaml:419]: ?

File: src/Files.App/UserControls/Widgets/DrivesWidget.xaml [hunk 3]
```diff
@@ -9,7 +9,7 @@
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	mc:Ignorable="d">
 
-	<Grid>
+	<Grid Padding="16,0,16,0">
```
Comment (yaira2) [inline src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: ?

File: src/Files.App/UserControls/AddressToolbar.xaml (hunk 2 above)
Comment (0x5bfa) [reply src/Files.App/UserControls/AddressToolbar.xaml:419]: I couldn't build without removing this state group.

Comment (yaira2) [reply src/Files.App/UserControls/AddressToolbar.xaml:419]: It shouldn't have anything to do with this PR, are you getting the exception in `main` as well?

Comment (0x5bfa) [reply src/Files.App/UserControls/AddressToolbar.xaml:419]: Let me revert and see what will happened.

File: src/Files.App/UserControls/Widgets/DrivesWidget.xaml (hunk 3 above)
Comment (yaira2) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: I think it's better to fix this on the ListView as opposed to making changes on the Expander

File: src/Files.App/UserControls/AddressToolbar.xaml (hunk 2 above)
Comment (yaira2) [reply src/Files.App/UserControls/AddressToolbar.xaml:419]: Did it work?

File: src/Files.App/UserControls/Widgets/DrivesWidget.xaml (hunk 3 above)
Comment (0x5bfa) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: then I have to set negative margins on the ListView.

Comment (yaira2) [reply src/Files.App/UserControls/Widgets/DrivesWidget.xaml:12]: Isn't that what we did before?

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**